modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/configuration/FunctionActionExecutorImplConfiguration.java

```diff
@@ -12,14 +12,18 @@
  */
 @Meta.OCD(
 	factory = true,
-	id = "com.liferay.portal.workflow.kaleo.runtime.internal.configuration.FunctionActionExecutorImplConfiguration"
+	id = "com.liferay.portal.workflow.kaleo.runtime.internal.configuration.FunctionActionExecutorImplConfiguration",
+	localization = "content/Language",
+	name = "function-action-executor-impl-configuration-name"
 )
 public interface FunctionActionExecutorImplConfiguration {
 
-	@Meta.AD
+	@Meta.AD(
+		name = "oauth2-application-external-reference-code", required = false
+	)
 	public String oAuth2ApplicationExternalReferenceCode();
 
-	@Meta.AD
+	@Meta.AD(name = "resource-path", required = false)
 	public String resourcePath();
 
```

Comment:
@AkramYoussoufi, for configuration can't display in UI, I suggest using `@ExtendedObjectClassDefinition(generateUI = false)` for these Configuration class, instead of adding language key. Please refer to AssetPublisherSelectionStyleConfiguration.